src/main/java/ch/njol/skript/expressions/ExprRepeat.java

```diff
@@ -0,0 +1,87 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Name("Repeat String")
+@Description("Repeats inputted strings a given amount of times.")
+@Examples({
+	"broadcast nl and nl repeated 200 times",
+	"broadcast \"Hello World \" repeated 5 times"
+})
+@Since("INSERT VERSION")
+public class ExprRepeat extends SimpleExpression<String> {
+
+	static {
+		Skript.registerExpression(ExprRepeat.class, String.class, ExpressionType.COMBINED, "%strings% repeated %integer% times");
+	}
+
+	private Expression<String> strings;
+	private Expression<Integer> repeatCount;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		strings = (Expression<String>) exprs[0];
+		repeatCount = (Expression<Integer>) exprs[1];
+		return true;
+	}
+
+	@Override
+	protected @Nullable String[] get(Event event) {
+		int repeatCount = this.repeatCount.getOptionalSingle(event).orElse(0);
+		if (repeatCount < 1)
+			return new String[0];
+		List<String> stringList = Arrays.stream(strings.getArray(event))
+			.map(string -> StringUtils.multiply(string, repeatCount)).collect(Collectors.toList());
+		return stringList.toArray(new String[0]);
```

Comment:
You're doing to much here. Expressions have a `stream` method you can be utilizing.
```suggestion
		return strings.stream(event)
				.map(string -> StringUtils.multiply(string, repeatCount))
				.toArray(String[]::new);
```

Reply:
Will fix in 1-2 hours still stuck at work wasn't aware of this